src/OAuth2/Storage/Phalcon/Phalcon.php

```diff
@@ -664,4 +685,204 @@ public function getBuildSql($dbName = 'oauth2_server_php')
         return $sql;
     }
 
+}
+
+
+/**
+ * Class PhalconConf
+ * @package OAuth2\Storage\Phalcon
+ *
+ * This config class is used purely internally for communication
+ * between the model files and the main Phalcon storage class
+ * for setting table names dynamically.
+ */
+class PhalconConf
```

Comment:
@bshaffer I was trying to do that, but I ran into a problem. It's really due to the nature of Phalcon.

Phalcon projects use a Dependency Injector to manage data interactions between model classes and everything else. The reason for this is that when calling model classes, its is done statically. That means there is no way to pass table names in a one-stop-shop fashion to the models unless you do it every time you statically call the `Model::find($tableName, $parameters)` or something like that. The way I accomplished this is to take a Dependency Injector `$di` and add a custom service to it and pass it on to all the models once. This way, all the models can just reference that service in the `$di` when they need to lookup table maps. The catch, though, is that a `$di` service must be an instance of a class, ANY class, so I just made a holder class to hold the information while passing to the models. 

It's not the most elegant way, but if you can find a better solution I'd seriously love to use it because I know this is the ugliest way to accomplish this task.